core/src/epicli/data/common/ansible/playbooks/roles/repository/files/download-requirements/ubuntu-18.04/common.sh

```diff
@@ -109,18 +115,15 @@ print_array_as_shell_escaped_string() {
 
 run_cmd() {
     local -a cmd_arr=("$@")
-
-    local output
-    output=$(print_array_as_shell_escaped_string "${cmd_arr[@]}")
-    echo "run:" "$output"
+    local output=$(print_array_as_shell_escaped_string "${cmd_arr[@]}")
```

Comment:
I would keep the original code to detect issue with `print_array_as_shell_escaped_string`.